Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -408,10 +408,6 @@ void ADefiWalletCoreActor::InitializeNewDevelopmentOnlyWallet(
     rust::cxxbridge1::String result =
         _coreWallet->get_address(CoinType::CryptoOrgMainnet, 0);
     output = UTF8_TO_TCHAR(result.c_str());
-    // zeroize mnemonics
-
-    zeroize_buffer((char *)result.data(), 0, result.size());
```

Comment:
why not to zeroize when not in use?

Reply:
It is misleading because the `result` here is the `address` instead of `mnemonics`.